src/contracts/misc/GhoCcipSteward.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {IGhoCcipSteward} from './interfaces/IGhoCcipSteward.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+import {UpgradeableLockReleaseTokenPool} from './deps/Dependencies.sol';
+import {RateLimiter} from './deps/RateLimiter.sol';
+
+/**
+ * @title GhoCcipSteward
+ * @author Aave Labs
+ * @notice Helper contract for managing parameters of the CCIP token pools
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ * @dev Requires roles RateLimitAdmin and BridgeLimitAdmin (if on Ethereum) on GhoTokenPool
+ */
+contract GhoCcipSteward is RiskCouncilControlled, IGhoCcipSteward {
+  /// @inheritdoc IGhoCcipSteward
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IGhoCcipSteward
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IGhoCcipSteward
+  address public immutable GHO_TOKEN_POOL;
+
+  /// @inheritdoc IGhoCcipSteward
+  bool public BRIDGE_LIMIT_ENABLED;
+
+  /**
+   * @dev Only methods that are not timelocked can be called if marked by this modifier.
+   */
+  modifier notTimelocked(uint40 timelock) {
```

Comment:
this must be applied to all functions, please check

Reply:
Done